src/GuestConfiguration/GuestConfiguration/Models/PolicyData.cs

```diff
@@ -41,6 +42,8 @@ public PolicyData()
 
         }
 
+        public string PolicySetDefinitionName { get; set; }
```

Comment:
Also, please add a changelog entriy describing these changes - this is the trigger that determines when a new module will be released.

Reply:
Updated